tests/core/resource/test_model.py

```diff
@@ -64,6 +64,13 @@ def test_delete_supports_collection_filters(self):
         records = result['data']
         self.assertEqual(len(records), 1)
 
+    def test_delete_collection_gives_number_of_deletable_records_in_headers(self):
+        self.resource.request.GET = {'_limit': '1'}
+        self.resource.collection_delete()
+        headers = self.last_response.headers
+        count = headers['Total-Records']
+        self.assertEquals(int(count), 2)
```

Comment:
I agree, i've included it there because there's a pretty similar test to check `Total-Records` on get operations. Maybe we should move both.

https://github.com/gabisurita/kinto/blob/4ba9cb6a5dacc015b39754a4b4f30dc26ec55a5e/tests/core/resource/test_model.py#L11